recipes/xtensor/all/conanfile.py

```diff
@@ -0,0 +1,70 @@
+import os
+
+from conans import CMake, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class XtensorConan(ConanFile):
+    name = "xtensor"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xtensor-stack/xtensor"
+    description = "C++ tensors with broadcasting and lazy computing"
+    topics = ("conan", "numpy", "multidimensional-arrays", "tensors")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "xsimd": [True, False],
+        "tbb": [True, False],
+        "openmp": [True, False],
+    }
+    default_options = {"xsimd": True, "tbb": False, "openmp": False}
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_folder = self.name + "-" + self.version
+        os.rename(extracted_folder, self._source_subfolder)
+
+    def configure(self):
+        if self.options.tbb and self.options.openmp:
+            raise ConanInvalidConfiguration(
+                "The options 'tbb' and 'openmp' can not be used together."
+            )
+
+        if self.settings.compiler == "gcc":
+            v = tools.Version(str(self.settings.compiler.version))
+            if v < "6.0":
+                raise ConanInvalidConfiguration("gcc >= 6.0 required")
+
+        if self.settings.compiler in ["gcc", "clang"]:
+            if str(self.settings.compiler.libcxx) == "libstdc++":
+                self.settings.compiler.libcxx = "libstdc++11"
```

Comment:
You can't force a setting in the recipe, otherwise it will cause an unexpected behavior.

Reply:
Yes, It's wrong to set the recipe by force. I will correct it.